app/views/carts/index.html.erb

```diff
@@ -23,7 +23,14 @@
 </div>
 <div class="checkout">
     <% if current_user %>
-    <%= link_to "Checkout", new_order_path(current_user) %>
+      <%= form_tag do |t| %>
+
+      <%= label_tag :address %>
+      <%= collection_select :user, :addresses, Address.where(user_id: current_user.id), :id, :street %>
+
+      <%= submit_tag("Checkout") %>
+      <% end %>
+      
```

Comment:
As we approach the deadline for this project, let's start styling as we're coding -- even if it's just wrapping these around a `<ul>` and `<li>` tags.

Reply:
I'm also not 100% sure what line 29 is doing. Are we creating a drop down list of addresses belonging to the user? What is `:id` and `:street` doing?

Reply:
Yeah it is creating a drop down selection of address belonging to the current user. The user will then select one of their addresses before submitting their order. The address then get's attached to the order through the complete_address method. :id organizes the selection based on the address ids, whereas :street displays the addess' street so that users can select based on street.